packages/angular/projects/clr-angular/src/data/datagrid/datagrid.spec.ts

```diff
@@ -673,6 +718,43 @@ export default function (): void {
         const rows = this.context.clarityElement.querySelectorAll('.datagrid-cell');
         expect(['1', '1', '2', '4', '3', '9']).toEqual([...rows].map(r => r.textContent));
       });
+
+      it('sorting by column works when pagination is also added', function () {
+        // Instantiate the component only to obtain the test data inside
+        const componentInstance = new SortPaginationTest();
+        const testUsers = componentInstance.users;
+        const sortedTestUsers = [...testUsers].sort((a, b) => a.name.localeCompare(b.name));
```

Comment:
There is no need for `testUsers`  as constant we are making a copy here `[...testUsers]`

```js
const sortedTestUsers = [...componentInstance.users].sort((a, b) => a.name.localeCompare(b.name));
const reverseSortedTestUsers = [...sortedTestUsers].reverse();
```

Reply:
`testUsers` is used on line 731 to check if the initial data is visualized correctly.